pkg/manifests/manifests.go

```diff
@@ -174,6 +187,38 @@ func MetricsRoleBinding() *rbacv1.RoleBinding {
 	return rb
 }
 
+func CanaryNamespace() *corev1.Namespace {
+	ns, err := NewNamespace(MustAssetReader(CanaryNamespaceAsset))
+	if err != nil {
+		panic(err)
```

Comment:
Adding panics makes things a little harder to test. Can these not return error?

Reply:
These new functions follow along with all of the other asset creation functions. IMO, we should panic here since being unable to read any of the operator's resources in via `assets` is very concerning.